### PR TITLE
Feature 4

### PR DESCRIPTION
**Pull request by withlovedese**
Added the comment feature and a button that shows the comment

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -15,18 +15,65 @@
...
+
+    // Redirect back to the HTML page.
+    response.sendRedirect("/index.html");
+  }
+   /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:70]: Nice work with null check

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -15,18 +15,65 @@
...
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+
+    // Break the text into individual words.
+    String[] words = text.split("\\s*,\\s*");
+
+    // Respond with the result.
     response.setContentType("text/html;");
-    response.getWriter().println("Hello Dese!");
+    myComments.add(text);
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: There are different scopes into which we add objects. Request/Response scope only stays for a particular request. Did this step help. Are we trying to use mycomments object. Else we might not need to add it.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -22,9 +22,20 @@ <h1>Adesemi Chloe Elumaro</h1>
...
+      <form action="/data" method="POST">
+      
+      <p>Leave me a comment or let me know what you think of my portfolio!:</p>
+      <textarea name="text-input">Name,comment</textarea>
+      <br/><br/>
+      <input type="submit" />
+      </form>
+
+      <p> Click to see previous comments!<p>
+      <button onclick="getCommentUsingAsyncAwait()">Comments</button>
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/webapp/index.html:35]: Do you get comments by a button click. Seems like it makes a new request and returns an empty arraylist from the get call

**Review by ReshmaMary-gcloud — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (withlovedese) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: I use the myComments  object to temporarily store the comments

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (withlovedese) [reply portfolio/src/main/webapp/index.html:35]: Clicking on the button shows the comments stored in myComments I believe

**Comment by ReshmaMary-gcloud** [comment]
Thank you for clarifying the comments. Make sure you test out your changes.

**Pull request merged**